pom.xml

```diff
@@ -47,6 +47,24 @@
     <module>it</module>
   </modules>
 
+  <repositories>
+    <repository>
+      <id>central</id>
+      <url>http://repo1.maven.org/maven2</url>
```

Comment:
What's the purpose of this change? If done to resolve HTTP access failures from sonatype, shouldn't it be `https`?

Reply:
Good catch.

> If done to resolve HTTP access failures from sonatype, shouldn't it be https?

No, this line is to prevent SNAPSHOT versions being downloaded from the repository. i.e. JKube must be built locally to be tested (PR integration tests will fail if JKube has not been built)